lib/checks/aria/invalidrole.js

```diff
@@ -1 +1,3 @@
-return !axe.commons.aria.isValidRole(node.getAttribute('role'));
+return !axe.commons.aria.isValidRole(node.getAttribute('role'), {
+	allowAbstract: true
```

Comment:
The word `allow` is throwing me off here, like abstract roles are being "permitted" rather than "included for evaluation". Maybe that's just me though?